stable/rabbitmq-ha/values.yaml

```diff
@@ -315,7 +315,7 @@ terminationGracePeriodSeconds: 10
 
 service:
   annotations: {}
-  clusterIP: None
+  # clusterIP:
```

Comment:
I would leave `clusterIP`,  without the comment

Reply:
Reverting to the original value after @steven-sheehy 's comment